js/hoot/control/utilities/ImportDataset.js

```diff
@@ -454,7 +454,7 @@ Hoot.control.utilities.importdataset = function(context) {
         } else if(typeName === 'OSM') {
             d3.select('#ingestfileuploader')
             .property('multiple', 'false')
-            .attr('accept', '.osm')
+            .attr('accept', '.osm,.osm.zip')
```

Comment:
I think only one of Firefox and Chrome actually enforces the accept values.  But in this case, we are mandating the zipped osm file be foo.osm.zip only?  And won't support foo.zip that contains an osm file?